specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/examples/WafPolicyCreateOrUpdate.json

```diff
@@ -159,7 +177,23 @@
             "fileUploadLimitInMb": 4000,
             "requestBodyCheck": true,
             "customBlockResponseStatusCode": 405,
-            "customBlockResponseBody": "SGVsbG8="
+            "customBlockResponseBody": "SGVsbG8=",
+            "logScrubbing": {
+              "state": "Enabled",
+              "scrubbingRules": [
+                {
+                  "state": "Enabled",
+                  "matchVariable": "RequestArgNames",
+                  "selectorMatchOperator": "Equals",
+                  "selector": "test"
+                },
+                {
+                  "matchVariable": "RequestIPAddress",
```

Comment:
"state" not needed?

Reply:
Default value is "Enabled" if it is not specified. Let me add state as well.

Reply:
Missed adding state here?